dashboard/src/containers/AppNewContainer/AppNewContainer.tsx

```diff
@@ -45,7 +49,7 @@ function mapDispatchToProps(dispatch: Dispatch<IStoreState>) {
       dispatch(actions.charts.getChartValues(id, version)),
     getChartVersion: (id: string, version: string) =>
       dispatch(actions.charts.getChartVersion(id, version)),
-    push: (location: string) => dispatch(push(location)),
+    push: (location: string) => dispatch(push(location) as any),
```

Comment:
why it's needed to set `push(location) as any`?

Reply:
It is the only way I managed to make it work. I tried different things like `RouterAction` but no luck. It seems that push does not fit neither Dispatch nor ThunkDispatch. 

In any case, I'll look into it tomorrow, there must be a better way to do this. Thanks!

Reply:
I fixed it. Initially was not working because I was using type intersection instead of union. But in the end I decided to use the top level redux `Action` type for simplicity. https://github.com/kubeapps/kubeapps/pull/650#discussion_r218931399